src/libConsensus/ConsensusLeader.cpp

```diff
@@ -210,7 +210,9 @@ void ConsensusLeader::GenerateConsensusSubsets() {
       }
     }
 
-    random_shuffle(peersWhoCommitted.begin(), peersWhoCommitted.end());
+    std::random_device randomDevice;
+    std::mt19937 randomEngine(randomDevice());
+    shuffle(peersWhoCommitted.begin(), peersWhoCommitted.end(), randomEngine);
```

Comment:
is this because of random_shuffle deprecation?

Reply:
Indeed.